OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -86,6 +86,28 @@ public void PlayMusic(string track = null, LuaFunction func = null)
 				playlist.Play(musicInfo);
 		}
 
+		[Desc("Play track defined in music.yaml as background music." +
```

Comment:
"Play track defined in music.yaml or map.yaml as background music."

Now that #9488 has been merged, `map.yaml` can include music track(s) for that map.